servers.json

```diff
@@ -1,4 +1,10 @@
-[ 
+[   {
```

Comment:
Please drop the second bracket to a new line.

Reply:
Changed!